Application/Validations/TraderValidator.cs

```diff
@@ -0,0 +1,22 @@
+using Domain.Entities;
+using FluentValidation;
+
+namespace Application.Validations
+{
+    public class TraderValidator : AbstractValidator<Trader>
+    {
+        public TraderValidator()
```

Comment:
The validation should be on the request and not on the Trader object. and use it in the service or explain why you put it in the controller.